client/src/components/HomeScreen.test.tsx

```diff
@@ -117,8 +115,18 @@ describe('Home screen', () => {
         isMultiJurisdiction: true,
       }),
       ...setupScreenCalls,
+      aaApiCalls.getJurisdictionFile,
+      aaApiCalls.getRounds,
+      ...setupScreenCalls,
+      aaApiCalls.getSettings(auditSettings.blank),
+      aaApiCalls.getJurisdictionFile,
```

Comment:
any way this chunk can go inside the definition `setupScreenCalls`? It looks like it's repeated three times in this PR

Reply:
@jonahkagan I was waffling on that. Here's why:

```
      ...setupScreenCalls,
      aaApiCalls.getSettings(auditSettings.blank),
      aaApiCalls.getJurisdictionFile,
```
You see this pattern about as much as you see this pattern:

```
      ...setupScreenCalls,
      aaApiCalls.getJurisdictionFile,
```

You also may or may not have the `getRounds` bit after, too. So basically, if I'm going to include even one more line, whether it's `getSettings` or `getJurisdictionFile`, I'd need to create two separate versions of `setupScreenCalls` or have some other complexity to include or exclude the settings endpoint. This looked to me like the simplest solution while still avoiding as much duplication as possible.

Reply:
cool, seems like this is fine then. hopefully we can reduce the number of API calls as a better solution :)